### PR TITLE
Clarify that this isn't just for live demos!

### PR DESCRIPTION
**Pull request by RamblingCookieMonster**

### CODE REVIEW

**Comment by RamblingCookieMonster** [comment]
Coffee hasn't hit yet, hopefully that's clear enough!

File: README.md [hunk 1]
```diff
@@ -28,6 +28,21 @@ Include info the audience will find helpful:
...
+* Slides
+* Slides with PowerShell on them
+* The console
+* VS Code
+* PowerShell ISE
+* Pre-recorded video that you narrate
+* Notepad (maybe not)
+* Something else
+
+Just make sure you zoom enough for your audience to see
```
Comment (glennsarti) [inline README.md]: nit:  Zoom in so it just fits in the whole screen.  Don't be shy.. Bigger is better!  You can't ask the audience if they can see

Comment (michaeltlombardi) [reply README.md]: @RamblingCookieMonster once this language is updated I'm happy to stamp and merge. :)

Comment (RamblingCookieMonster) [reply README.md]: Good call!  Added, thanks!

**Pull request merged**